### ISSUE TITLE
Undefined variable Vdata

### ISSUE THREAD

**Issue by bjacog** [Tealium/tealium-react-native#26]
I'm implementing Tealium support for an app.

Ihave installed the iOS lbrary via Cocoapods and added the bridging files from my node_modules instead of here https://github.com/Tealium/tealium-react-native/tree/master/ios, since the files in the link seem quite outdated compared to the files that ship with `node_modules/tealium-react-native/ios/TeaiumModule/*`.

I get an error when trying to build for these lines:
```
@implementation RCTConvert (CollectURL)
RCT_ENUM_CONVERTER(TEALCollectURL, (@{
                                      @"vdata" : @(Vdata),
                                      @"collect" : @(Event),
                                      }), Event, integerValue)
@end
```

Saying that Vdata is not declared/defined.

Vdata is only used AFAICS in `initializeCustom` to set a collectURL if none was specified.

The only reference to Vdata I could find seems to be a link to a pixel-like gif file.
https://github.com/Tealium/tealium-ios/search?q=vdata&unscoped_q=vdata

```
NSString * const TealiumCollectBaseURL = @"https://collect.tealiumiq.com/vdata/i.gif?";
```

If I am not using `initializeCustom`, is it safe to comment out references to `Vdata`?

**Comment by craigrouse** [comment]
@bjacog yes, it should be safe to comment this out for now.

**Comment by jameskeith** [comment]
@bjacog - this should be resolved in the latest release (v1.0.7) 
please let us know if you have any further problems.
